### PR TITLE
[onert-micro] Introduce kernels without TF

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This PR removes TensorFlow dependence from onert-micro kernels.

for issue: https://github.com/Samsung/ONE/issues/10817
second part from draft: https://github.com/Samsung/ONE/pull/10816

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
@BalyshevArtem ptal build fail

**Comment by chunseoklee** [comment]
> > @BalyshevArtem ptal build fail
> 
> We discussed with @SlavikMIPT about it yesterday. This happens due to the lack of RAM in the test environment for compilation using arm-none-eabi, and kill process occurs. At the moment, we have decided only to restart the work, and wait for a successful completion.

You mean it happens when two jobs(Build onert-micro with arm-none-eabi-gcc and Run onert-micro Unit tests) run at the same time or it can happen regardless of other job ?

**Comment by BalyshevArtem** [comment]
> You mean it happens when two jobs(Build onert-micro with arm-none-eabi-gcc and Run onert-micro Unit tests) run at the same time or it can happen regardless of other job ?

The second one, it can happen regardless of other job

File: onert-micro/luci-interpreter/src/kernels/Tanh.cpp [hunk 1]
```diff
@@ -119,20 +121,23 @@ void evalInteger(const circle::Tensor *input, const circle::Tensor *output,
   auto *output_data = runtime_graph->getDataByTensor(output);
   assert(output_data);
 
+  const int flat_size = kernels::getTensorRuntimeShape(input, runtime_graph).flatSize();
+
   const auto input_dtype = Tensor::element_type(input);
   switch (input_dtype)
   {
+    // TODO: enable it
+#if 0
```
[the remark this answers is not in the document]
Comment (BalyshevArtem) [reply onert-micro/luci-interpreter/src/kernels/Tanh.cpp:130]: I haven't checked it yet, that is why I am not sure in this part, so need double check it and than we can enable it.

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**